components/inputs/input-time.js

```diff
@@ -426,5 +438,18 @@ class InputTime extends SkeletonMixin(FormElementMixin(LitElement)) {
 			{ bubbles: true, composed: false }
 		));
 	}
+
+	async _open() {
+		if (this.disabled || this.skeleton) return;
+		if (!this._dropdownFirstOpened) this._dropdownFirstOpened = true;
+
+		const dropdown = this.shadowRoot.querySelector('d2l-dropdown-menu');
+		if (dropdown && dropdown.opened) return;
+
+		await this.updateComplete;
+
+		dropdown.open(true);
+		this.shadowRoot.querySelector('d2l-menu').focus();
+	}
```

Comment:
I'm curious... instead of `_open()` and `_close()` could we bind to `d2l-dropdown-menu`'s `opened` in `render()`? Something like:

```javascript
const opened = this.opened && !this.disabled && !this.skeleton;
...
return html`...<d2l-dropdown-menu ?opened="${opened}">...`;
```

Reply:
ooo good idea, I think that works!